SignalServiceKit/src/Messages/OWSMessageReceiver.m

```diff
@@ -260,14 +250,24 @@ - (void)databaseViewRegistrationComplete
 
 #pragma mark - instance methods
 
+- (dispatch_queue_t)serialQueue
```

Comment:
Here too there are real perf wins to using a dedicated serial queue instead of the main queue.